src/js/components/FormField/FormField.js

```diff
@@ -229,6 +230,19 @@ const FormField = forwardRef(
               focusIndicator: false,
```

Comment:
```suggestion
              focusIndicator: false,
              pad: 'CheckBox'.indexOf(child.type.displayName) !== -1 ? formFieldTheme?.checkBox?.pad : undefined,
```

Does doing this work? And then removing the new 'if' statement below? Just trying to minimize repeated logic if possible

Reply:
let me test on my side!

Reply:
worked good!